src/HearThis/Script/Project.cs

```diff
@@ -303,24 +300,10 @@ public void ClearAllSkippedBlocks()
 			_scriptProvider.ClearAllSkippedBlocks(Books);
 		}
 
-		public void BackUpRecordingForSkippedLine()
-		{
-			var recordingPath = GetPathToRecordingForSelectedLine();
-			File.Move(recordingPath, Path.ChangeExtension(recordingPath, "skip"));
-		}
-
 		private void OnScriptBlockUnskipped(IScriptProvider sender, int bookNumber, int chapterNumber, ScriptLine scriptBlock)
 		{
-			var recordingPath = ClipRepository.GetPathToLineRecording(
-				Name, Books[bookNumber].Name, chapterNumber, scriptBlock.Number - 1);
-			var skipPath = Path.ChangeExtension(recordingPath, "skip");
-			if (File.Exists(skipPath))
-			{
-				File.Move(skipPath, recordingPath);
-
-				if (OnScriptBlockRecordingRestored != null)
-					OnScriptBlockRecordingRestored(this, bookNumber, chapterNumber, scriptBlock);
-			}
+			if (ClipRepository.RestoreBackedUpClip(Name, Books[bookNumber].Name, chapterNumber, scriptBlock.Number - 1))
+				OnScriptBlockRecordingRestored?.Invoke(this, bookNumber, chapterNumber, scriptBlock);
```

Comment:
Feels as though there should be something parallel to handle the event of turning on the block being skipped and back it up. If there's a good reason to handle that in a different place (or some reason it's not needed), a comment would be nice.